examples/big.rs

```diff
@@ -0,0 +1,25 @@
+extern crate term_grid;
```

Comment:
maybe add a license too ?

Reply:
and reference this example in the readme ?

Reply:
Sounds good. How do you want to phrase it? This is the format from coreutils, but maybe it's weird to say this is part of the uutils coreutils package. We should probably also give credit here to the original author.

```
// This file is part of the uutils coreutils package.
//
// (c) author <email>
//
// For the full copyright and license information, please view the LICENSE
// file that was distributed with this source code.

Reply:
just keep the last 2 lines :)